.claude/profiles/personal/projects/-home-das-Downloads-NemoClaw/memory/user_australian.md

```
@@ -0,0 +1,7 @@
+---
+name: Australian greeting preference
+description: User is Australian and prefers "G'day" over "Hi" in messages
+type: user
+---
+
+User is Australian. Prefers "G'day" as a greeting rather than "Hi" when drafting messages or comments on their behalf.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)